src/java.xml/share/classes/javax/xml/xpath/XPathFactory.java

```diff
@@ -369,4 +370,57 @@ public abstract boolean getFeature(String name)
     * @return New instance of an <code>XPath</code>.
     */
     public abstract XPath newXPath();
+
+    /**
+     * Sets a property for this {@code XPathFactory}. A property set on the
+     * factory is effective to the {@code XPath} object created thereafter, but
+     * not ones that may have been created beforehand.
```

Comment:
It might be clear to say something like "The property has no impact on XPath object that are already created".

Reply:
Thanks Alan. Updated accordingly.